### DISCUSSION TITLE
Cloud-BB and PostgreSQL in Docker

### DISCUSSION THREAD

**Discussion by svobbl** [Budibase/budibase#14165] [Help] [answered]
Hi everyone,

I am pretty new to programmingl, would like to get started and set up a simple project.
For this I need BB app, with which I can write data in PostgreSQL DB. I've been struggling with it for 3 days now, learned a lot, but can't get it to work. When I want to connect BB app with postgresql DB, I get ECONNREFUSED message. 

A general question first, before I describe all my attempts: is it even possible to connect Cloud-BB with local Docker on my PC or i have to have BB in Docker as well?

Please help!

**Comment by JayP718** [comment]
You need to have them both in docker or you can host Postgres somewhere in the cloud

**Reply by svobbl** [reply]
Hi, thank you!
Do you mean somewhere like VPS? Or what else is possible in that case?

**Reply by ConorWebb96** [reply, marked as answer]
You can use something like `ingrok` to connect to a local database. You won't be able to connect to a database unless the database's connection is publicly available.

Hopefully this makes sense.
